diff --git a/node_modules/@digitalcredentials/jsonld/lib/context.js b/node_modules/@digitalcredentials/jsonld/lib/context.js
index 2d282f8..a400328 100644
--- a/node_modules/@digitalcredentials/jsonld/lib/context.js
+++ b/node_modules/@digitalcredentials/jsonld/lib/context.js
@@ -985,6 +985,10 @@ api.createTermDefinition = ({
     activeCtx.protected[term] = true;
     mapping.protected = true;
     if(!_deepCompare(previousMapping, mapping)) {
+      if(term === 'VerifiableCredential' || term === 'VerifiablePresentation') {
+        console.warn('WARNING: protected term redefinition', {term});
+        return;
+      }
       throw new JsonLdError(
         'Invalid JSON-LD syntax; tried to redefine a protected term.',
         'jsonld.SyntaxError',
