diff --git a/node_modules/@digitalcredentials/jsonld/lib/expand.js b/node_modules/@digitalcredentials/jsonld/lib/expand.js
index dc52fc6..be9f56e 100644
--- a/node_modules/@digitalcredentials/jsonld/lib/expand.js
+++ b/node_modules/@digitalcredentials/jsonld/lib/expand.js
@@ -475,6 +475,9 @@ async function _expandObject({
 
   for(const key of keys) {
     let value = element[key];
+    if(value === undefined) {
+      continue;
+    }
     let expandedValue;
 
     // skip @context
